introduction.md

```diff
@@ -8,6 +8,8 @@ as you know
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?